src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/help-inheritYamlMergeStrategy.html

```diff
@@ -0,0 +1 @@
+False by default
```

Comment:
Can you be more explicit on what this checkbox is doing?

Reply:
Yes, forgot to push last version :)